webpack.config.renderer.dev.js

```diff
@@ -50,6 +50,23 @@ export default merge.smart(baseConfig, {
   module: {
     rules: [
       {
+        test: /\.jsx?$/,
+        exclude: /node_modules/,
+        use: {
+          loader: 'babel-loader',
+          options: {
+            cacheDirectory: true,
+            plugins: [
```

Comment:
We should keep the `class-properties` before `es2015-classes` otherwise it will throw `Missing class properties transform` error. (plugins run before presets, we can see in [babel doc](https://babeljs.io/docs/plugins/#plugin-preset-ordering))